README.md

```diff
@@ -1,82 +1,86 @@
-    * **c**: Macro to make a lazy Iterator collection comprehensions, all other are base on this one.
-    * **cbheap**: Macro to `BinaryHeap` collection comprehensions
-    * **cbtmap**: Macro to `BTreeMap` collection comprehensions
-    * **cbtset**: Macro to `BTreeSet` collection comprehensions
-    * **cdeque**: Macro to `VecDeque` collection comprehensions
-    * **cmap**: Macro to `HashMap` collection comprehensions
-    * **cset**: Macro to `HashSet` collection comprehensions
-    * **cvec**: Macro to `Vec` collection comprehensions
+    * **c**: Macro to make a lazy Iterator collection comprehensions, all other are
+      based on this one.
+    * **cbheap**: Macro to [`BinaryHeap`] collect() comprehensions
+    * **cbtmap**: Macro to [`BTreeMap`] collect() comprehensions
+    * **cbtset**: Macro to [`BTreeSet`] collect() comprehensions
+    * **cdeque**: Macro to [`VecDeque`] collect() comprehensions
+    * **cmap**: Macro to [`HashMap`] collect() comprehensions
+    * **cset**: Macro to [`HashSet`] collect() comprehensions
+    * **cvec**: Macro to [`Vec`] collect() comprehensions
  * **Smart Pointers:**
-    * **arc**: A simple macro to make a new `Arc` value.²
-    * **boxed**: A simple macro to make a new `Box` value.²
-    * **cell**: A simple macro to make a new `Cell` value.²
-    * **cow**: A simple macro to make a new `Cow` value.
-    * **mutex**: A simple macro to make a new `Mutex` value.²
-    * **refcell**: A simple macro to make a new `RefCell` value.²
-    * **rc**: A simple macro to make a new `Rc` value.²
+    * **arc**: A simple macro to make a new [`Arc`] value.²
+    * **boxed**: A simple macro to make a new [`Box`] value.²
+    * **cell**: A simple macro to make a new [`Cell`] value.²
+    * **cow**: A simple macro to make a new [`Cow`] value.
+    * **mutex**: A simple macro to make a new [`Mutex`] value.²
+    * **refcell**: A simple macro to make a new [`RefCell`] value.²
+    * **rc**: A simple macro to make a new [`Rc`] value.²
     * **rwlock**: A simple macro to make a new [`RwLock`] value.²
  * **Time/Duration:**
-    * **dur**: Creates a `Duration` object following a time pattern¹
+    * **dur**: Creates a [`Duration`] object following a time pattern¹
     * **sleep**: Makes a thread sleep a amount following a time pattern¹
     * **time**: Print out the time it took to execute a given expression in seconds
 
  1. A time pattern can be: mim, sec, nano, micro, milli
  2. Also can return a tuple if is given more than one parameter
 
 ## Examples
-**arc**, **boxed**, **cell**, **cow**, **mutex** and **refcell**: Usage are mostly the same, just change de Types and macros
+**arc**, **boxed**, **cell**, **cow**, **mutex** and **refcell**: Usage are mostly the
+same, just change de Types and macros
 ```rust
+use sugars::boxed;
 assert_eq!(Box::new(10), boxed!(10));
 ```
 
 **hmap** and **btmap**: Usage are the same, just change de Types and macros
 ```rust
+use std::collections::HashMap;
+use sugars::hmap;
+
```

Comment:
I'm not a huge fan of this `use` lines inside of the README, I would suggest removing those.

But they are present inside of the `lib.rs` file, so maybe we could have this difference between them, also because `lib.rs` docs comments are tested.

So, what do you think? Should I remove it from the README and leave it as it is in the `lib.rs` file?

Reply:
I dunno, in one hand it does make the usage clear and easy to copy-and-paste to test, on the other hand it takes more lines

Maybe a good compromise would be using the full namespace in the code instead of use statements.

Reply:
Nice, I'll do it when I get the time to.